decidim-debates/lib/decidim/debates/debate_serializer.rb

```diff
@@ -26,10 +26,7 @@ def serialize
           start_time: debate.start_time,
           end_time: debate.end_time,
           information_updates: debate.information_updates,
-          taxonomies: {
-            id: debate.taxonomies.map(&:id),
-            name: debate.taxonomies.map(&:name)
-          },
+          taxonomies: serialize_taxonomies(debate),
```

Comment:
My previous comments
```suggestion
          taxonomies: ,
```